src/gei/Handlers/MigrateRepoCommandHandler.cs

```diff
@@ -124,22 +124,24 @@ public async Task Handle(MigrateRepoCommandArgs args)
             return;
         }
 
-        var (migrationState, _, failureReason) = await _targetGithubApi.GetMigration(migrationId);
+        var (migrationState, _, failureReason, url) = await _targetGithubApi.GetMigration(migrationId);
 
         while (RepositoryMigrationStatus.IsPending(migrationState))
         {
             _log.LogInformation($"Migration in progress (ID: {migrationId}). State: {migrationState}. Waiting 10 seconds...");
             await Task.Delay(10000);
-            (migrationState, _, failureReason) = await _targetGithubApi.GetMigration(migrationId);
+            (migrationState, _, failureReason, url) = await _targetGithubApi.GetMigration(migrationId);
         }
 
         if (RepositoryMigrationStatus.IsFailed(migrationState))
         {
             _log.LogError($"Migration Failed. Migration ID: {migrationId}");
+            _log.LogInformation($"Migration log available at {url} or by running 'gh {CliContext.RootCommand} download-logs --github-target-org {args.GithubTargetOrg} --target-repo {args.TargetRepo}'");
```

Comment:
```suggestion
            _log.LogInformation($"Migration log available at {migrationLogUrl} or by running 'gh {CliContext.RootCommand} download-logs --github-target-org {args.GithubTargetOrg} --target-repo {args.TargetRepo}'");
```

Reply:
Resolved in b8273b92baf683a9429245d93d66d735d24127a4 :tada: